File: src/main/java/com/example/demo/map/TaskMapper.java
package com.example.demo.map;

import com.example.demo.po.Task;
import org.springframework.jdbc.core.RowMapper;

import java.sql.ResultSet;
import java.sql.SQLException;

public class TaskMapper implements RowMapper<Task> {
    @Override
    public Task mapRow(ResultSet rs, int i) throws SQLException {
        Task tsak = new Task();
        tsak.setId(rs.getInt("id"));
        tsak.setTitle(rs.getString("title"));
        tsak.setTask_describe(rs.getString("task_describe"));
        tsak.setPictures( rs.getString("pictures"));
        tsak.setCreate_uid(rs.getInt("create_uid"));
        tsak.setBrowsing_volume(rs.getInt("browsing_volume"));
        tsak.setCreate_time(rs.getTimestamp("create_time"));
        tsak.setEnd_time(rs.getTimestamp("end_time"));
        tsak.setStatus(rs.getInt("status"));
        return tsak;
    }
}

File: src/main/java/com/example/demo/map/TaskMapper.kt
package com.example.demo.map

import com.example.demo.po.Task
import org.springframework.jdbc.core.RowMapper
import java.sql.ResultSet
import java.sql.SQLException


class TaskMapper : RowMapper<Task> {
    @Throws(SQLException::class)
    override fun mapRow(rs: ResultSet, rowNum: Int): Task {
        val tsak = Task()
        tsak.id = rs.getInt("id")
        tsak.title = rs.getString("title")
        tsak.task_describe = rs.getString("task_describe")
        tsak.pictures = rs.getString("pictures")
        tsak.create_uid = rs.getInt("create_uid")
        tsak.browsing_volume = rs.getInt("browsing_volume")
        tsak.create_time = rs.getTimestamp("create_time")
        tsak.end_time = rs.getTimestamp("end_time")
        tsak.status = rs.getInt("status")
        return tsak
    }
}